### PR TITLE
fix: nodejs version

### PR DESCRIPTION
**Pull request by sugarshin**
- Fix Node.js version for GitHub Actions with the specified version
- Add .nvmrc

### CODE REVIEW

File: .github/workflows/deploy-to-netlify.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 'lts/*'
+          node-version: '18.14.2'
```
Comment (odanado) [inline .github/workflows/deploy-to-netlify.yml]: If you use `node-version-file`, you can reduce the description of the version to one time.

https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

Comment (sugarshin) [reply .github/workflows/deploy-to-netlify.yml]: I didn't know that. I will fix it. Thank you!

Comment (sugarshin) [reply .github/workflows/deploy-to-netlify.yml]: fixed: https://github.com/line/liff-playground/pull/19/commits/8fbb351ae5e4e5c355f04491436ea5b7a8f36a63

**Review by odanado — APPROVED** [review]
LGTM

**Review by cola119 — APPROVED** [review]

**Pull request merged**